src/drivers/dw/dma.c

```diff
@@ -831,17 +841,10 @@ static void dw_dma_verify_transfer(struct dma_chan_data *channel,
 
 		dcache_writeback_region(dw_chan->lli,
 					sizeof(struct dw_lli) * channel->desc_count);
-#else
-		while (lli->ctrl_hi & DW_CTLH_DONE(1)) {
-			lli->ctrl_hi &= ~DW_CTLH_DONE(1);
-			dw_chan->lli_current =
-				(struct dw_lli *)dw_chan->lli_current->llp;
-			lli = platform_dw_dma_lli_get(dw_chan->lli_current);
-		}
-#endif
 		break;
 	}
 #else
+
```

Comment:
spurious line

Reply:
corresponding to the line 829, an empty line between variables and codes.